### PR TITLE
Daffy staging

### PR DESCRIPTION
**Pull request by Tuxliri**

### CODE REVIEW

File: src/software-architecture/nodes.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
+# Nodes
+
+This section elaborates on all of the ROS nodes that run on your drone to make it fly autonomously. These are described in the order in which they appear in pi.screenrc.
+
+## \`0: rcore
```
Comment (afdaniele) [inline src/software-architecture/nodes.md]: This should be `roscore` (`!= rcore`)

File: src/software-architecture/nodes.md [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+Starts up a ROS master to allow the nodes to find each other.
+
+* Runs: `roscore`
+
+## \`1: FC
```
Comment (afdaniele) [inline src/software-architecture/nodes.md:10]: Why do we have the symbol \` in all the headers?

**Review by afdaniele — CHANGES_REQUESTED** [review]

File: src/software-architecture/nodes.md (hunk 2 above)
Comment (Tuxliri) [reply src/software-architecture/nodes.md:10]: It's to indicate that they are e.g. the Screen number 1 and you can access them with \`1. I can clarify it in the text.

Comment (afdaniele) [reply src/software-architecture/nodes.md:10]: Ping me here when done

Comment (Tuxliri) [reply src/software-architecture/nodes.md:10]: @afdaniele I added a clarification in 01932d0

**Review by afdaniele — APPROVED** [review]

**Pull request merged**